sdk/scala-sdk-protobuf/src/test/scala/kalix/scalasdk/JsonSupportSpec.scala

```diff
@@ -38,6 +38,12 @@ class JsonSupportSpec extends AnyWordSpec with Matchers {
       any.typeUrl should ===(JavaJsonSupport.KALIX_JSON + classOf[MyJsonable].getName)
       JsonSupport.decodeJson[MyJsonable](any).field should ===("foo")
     }
+    "serialize and deserialize Akka Done class" in {
+      val done = Done
+      val any = JsonSupport.encodeJson(done)
+      any.typeUrl should ===(JavaJsonSupport.KALIX_JSON + Done.getClass.getName)
```

Comment:
can't use `classOf[MyJsonable].getName` because for the scala object the result is different than `Done.getClass.getName` (additional `$` at the end)